src/scheduler/server_info.cpp

```diff
@@ -644,6 +667,15 @@ query_server_info(status *pol, struct batch_status *server)
 	site_set_share_head(sinfo);
 #endif /* localmod 034 */
 
+	if (sinfo->node_group_key == NULL and get_num_servers() > 1) {
```

Comment:
I think this needs to be &&

Reply:
Yeah, it'll need to copy it, otherwise calendaring will all get messed up.